client/slack.go

```diff
@@ -114,6 +114,7 @@ type messageEvent struct {
 	Message         string `json:"message"`
 	Timestamp       string `json:"timestamp"`
 	ThreadTimestamp string `json:"threadTimestamp"`
+	Thread string `json:"thread"`
```

Comment:
Run `go fmt` in this file, or "Reformar code" if you are using Goland. It's a minor indentation issue

Reply:
thx,